docker/backend-dev/conf/Config2-deploy-test.pm

```diff
@@ -0,0 +1,78 @@
+# This file is part of Product Opener.
+#
+# Product Opener
+# Copyright (C) 2011-2021 Association Open Food Facts
+# Contact: contact@openfoodfacts.org
+# Address: 21 rue des Iles, 94100 Saint-Maur des Fossés, France
+#
+# Product Opener is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package ProductOpener::Config2;
+
+use utf8;
+use Modern::Perl '2017';
+use Exporter    qw< import >;
+
+BEGIN
+{
+	use vars       qw(@ISA @EXPORT_OK %EXPORT_TAGS);
+	@EXPORT_OK = qw(
+		$server_domain
+		@ssl_subdomains
+		$data_root
+		$www_root
+		$geolite2_path
+		$mongodb
+		$mongodb_host
+		$mongodb_timeout_ms
+		$memd_servers
+		$facebook_app_id
+	    $facebook_app_secret
+		$robotoff_url
+		%server_options
+	);
+	%EXPORT_TAGS = (all => [@EXPORT_OK]);
+}
+use vars @EXPORT_OK ; # no 'my' keyword for these
+
+# server constants
+$server_domain = "openfoodfacts.net";
```

Comment:
If we used an env variable here `ENV{PRODUCT_OPENER_DOMAIN} || 'openfoodfacts.localhost'`, we would not need this additional config file